app/src/main/java/com/teamfillin/fillin/presentation/filmroll/CurationAdapter.kt

```diff
@@ -0,0 +1,91 @@
+package com.teamfillin.fillin.presentation.filmroll
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.teamfillin.fillin.com.teamfillin.fillin.presentation.home.NEW_PHOTOS_TYPE
+import com.teamfillin.fillin.com.teamfillin.fillin.presentation.home.NewPhotosAdapter
+import com.teamfillin.fillin.core.view.setOnSingleClickListener
+import com.teamfillin.fillin.databinding.ItemCurationBinding
+import com.teamfillin.fillin.databinding.ItemCurationFirstBinding
+import com.teamfillin.fillin.databinding.ItemNewPhotosListBinding
+import com.teamfillin.fillin.databinding.ItemNextButtonBinding
+
+class CurationAdapter() :
+    ListAdapter<CurationInfo, RecyclerView.ViewHolder>(CurationDiffUtil()) {
+
+
+    class CurationInfoViewHolder(private val binding: ItemCurationFirstBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun bind(film: CurationInfo) {
+            Glide.with(binding.root)
+                .load(film.image)
+                .into(binding.ivCurationfirst)
+            binding.tvCuration.text = "따뜻한 사진을 \n 원한다면"
```

Comment:
```suggestion
            binding.tvCuration.text = "따뜻한 사진을\n원한다면"
```

이렇게 나오면

```
따뜻한 사진을
 원한다면
```

이런식으로 텍스트가 보여지겠쥬?